src/bun.js/bindings/bindings.cpp

```diff
@@ -3147,7 +3147,7 @@ void JSC__JSPromise__rejectOnNextTickWithHandled(JSC__JSPromise* promise, JSC__J
             globalObject->m_asyncContextData.get()->getInternalField(0),
             promise,
             value);
-        RETURN_IF_EXCEPTION(scope, void());
+        RETURN_IF_EXCEPTION(scope, );
```

Comment:
this doesn't syntax error?

Reply:
it does not
the space appeases the preprocessor for a second value
empty value makes `return value__;` emit `return;` which satisfies a void return